core/src/main/scala/kafka/server/KafkaServer.scala

```diff
@@ -134,6 +134,8 @@ class KafkaServer(
 
   var autoTopicCreationManager: AutoTopicCreationManager = null
 
+  var clientToControllerChannelManager: Option[BrokerToControllerChannelManager] = None
+
   var alterIsrManager: AlterIsrManager = null
```

Comment:
Hello, I have a question, should we also share the channel between alterIsrManager and autoCreationManager, furthermore, also share the same one with alterReplicaStateManager proposed in KIP-589.

Reply:
We are planning to consolidate into two channels eventually:
1. broker to controller channel
2. client to controller channel

here, auto topic creation and forwarding fall into the 2nd category, while AlterIsr would be the 1st category.

Reply:
Looks good, I will try to file a ticket to consolidate broker to controller channel. 😉
It seems that the `BrokerServer` also holds AutoTopicCreationManager and ForwardingManager, so it also need to be changed